raiden/tests/integration/test_recovery.py

```diff
@@ -235,3 +237,80 @@ def test_recovery_blockchain_events(raiden_network, token_addresses, network_wai
         RANGE_ALL_STATE_CHANGES
     )
     assert search_for_item(restarted_state_changes, ContractReceiveChannelClosed, {})
+
+
+@pytest.mark.parametrize("deposit", [2])
+@pytest.mark.parametrize("number_of_nodes", [2])
+def test_node_clears_pending_withdraw_transaction_after_channel_is_closed(
+    raiden_network, token_addresses, network_wait, number_of_nodes, retry_timeout
+):
+    """ A test case related to https://github.com/raiden-network/raiden/issues/4639
+    Where a node sends a withdraw transaction, is stopped before the transaction is completed.
+    Meanwhile, the partner node closes the channel so when the stopped node is back up, it tries to
+    execute the pending withdraw transaction and fails because the channel was closed.
+    Expected behaviour: Channel closed state change should cancel a withdraw transaction.
+    Actual behaviour: The channel closure isn't detected on recovery and
```

Comment:
Is "Actual behaviour" the right wording to use in a test? In the linked bug report it makes total sense, but when reading this test after the PR is merged, this won't be the actual behaviour.

Reply:
Changed that to "Buggy behaviour"